### PR TITLE
Update unofficial.md

### PR DESCRIPTION
**Pull request by kwfn**
Changing use of `practise` to the noun form `practice`.

### CODE REVIEW

**Review by glyn — CHANGES_REQUESTED** [review]
I'm afraid I'm from the UK and this is correct UK English. If you'd like to fix this for Americans, please re-phrase the description, e.g. to use "practice" as a noun.

**Comment by kwfn** [comment]
Comment updated.

File: src/unofficial.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Introductory:
 * [_Learn Rust the Dangerous Way_](http://cliffle.com/p/dangerust/)
 * [Learning Rust](https://quinedot.github.io/rust-learning/index.html) - a collection of resources by `quinedot`
 * [Learning Rust With Entirely Too Many Linked Lists](https://rust-unofficial.github.io/too-many-lists/)
-* [Rust By Practice](https://practice.rs/) - Practise Rust with challenging examples, exercises and projects
+* [Rust By Practice](https://practice.rs/) - Practice Rust with challenging examples, exercises and projects
```
Comment (glyn) [inline src/unofficial.md:15]: Er, "practice" here is still a verb, so this is the American spelling.
```suggestion
* [Rust By Practice](https://practice.rs/) - Rust practice through challenging examples, exercises and projects
```

**Review by glyn — APPROVED** [review]
Reworked to UK English compatible with US English.

**Comment by kwfn** [comment]
It's also that it matches the title of the book being linked. Rust is technically based on American English as far as compiler documentation. I'm Canadian, personally I just prefer consistency. Thanks!

**Comment by glyn** [comment]
Understood. I don't mind switching to US English if someone with an eye for that (such as you) could submit a suitable PR.

**Pull request merged**